src/con_duct/__main__.py

```diff
@@ -512,14 +512,14 @@ def from_argv(
             "-l",
             "--log_level",
             default=DEFAULT_LOG_LEVEL,
-            choices=("CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG"),
-            help="Log level from duct operation.",
+            choices=("CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG", "NONE"),
+            help="Log level from duct operation, use NONE to disable duct output to stderr.",
         )
         parser.add_argument(
             "-q",
             "--quiet",
             action="store_true",
-            help="Disable duct logging output (to stderr)",
+            help="[deprecated, use log level NONE] Disable duct logging output (to stderr)",
```

Comment:
it would take longer for people to notice. Ideally there should be some kind of DeprecationWarning if this option is used.  Could you please add one on `if quiet:` ?

Reply:
Unfortunately no, if --quiet, we have to actually be quiet :)

Reply:
hm, makes sense although rubs against my instinct here... ok for now